### PR TITLE
add test case for ConfigMojo

### PR DESCRIPTION
**Pull request by andxu**

### CODE REVIEW

File: azure-spring-cloud-maven-plugin/src/test/java/com/microsoft/azure/maven/spring/prompt/PromptWrapperTest.java [hunk 1]
```diff
@@ -193,6 +201,54 @@ public void testBadTemplate() throws Exception {
...
+
+        try {
+            wrapper.handle("testId1", true, "foo");
+            fail("Should throw InvalidConfigurationException");
+        } catch (InvalidConfigurationException ex) {
+            // expected
+        }
+
+        try {
+            wrapper.handle("testId1", true, "foo");
```
Comment (Flanker32) [inline azure-spring-cloud-maven-plugin/src/test/java/com/microsoft/azure/maven/spring/prompt/PromptWrapperTest.java]: Duplicate with the front one?

File: azure-spring-cloud-maven-plugin/src/test/resources/pom-1.xml [hunk 2]
```diff
@@ -26,7 +26,7 @@
 		<plugins>
 			<plugin>
 				<groupId>com.microsoft.azure</groupId>
-				<artifactId>azure-spring-maven-plugin</artifactId>
+				<artifactId>azure-spring-cloud-maven-plugin</artifactId>
 				<version>0.0.1-SNAPSHOT</version>
```
Comment (Flanker32) [inline azure-spring-cloud-maven-plugin/src/test/resources/pom-1.xml]: Shall we add a parent pom or something else to get the correspond spring maven plugin version? Or we need to update every file when bump version

Comment (andxu) [reply azure-spring-cloud-maven-plugin/src/test/resources/pom-1.xml]: It is not good to use variable for artifactId

File: azure-spring-cloud-maven-plugin/src/test/java/com/microsoft/azure/maven/spring/prompt/PromptWrapperTest.java (hunk 1 above)
Comment (andxu) [reply azure-spring-cloud-maven-plugin/src/test/java/com/microsoft/azure/maven/spring/prompt/PromptWrapperTest.java]: Will delete it

**Pull request merged**